packages/clerk-js/src/core/clerk.ts

```diff
@@ -187,7 +187,7 @@ export default class Clerk implements ClerkInterface {
   };
 
   public signOut: SignOut = async (callbackOrOptions?: SignOutCallback | SignOutOptions, options?: SignOutOptions) => {
-    if (!this.client || !this.session) {
+    if (!this.client || this.client.sessions.length === 0) {
       return;
     }
     const cb = typeof callbackOrOptions === 'function' ? callbackOrOptions : undefined;
```

Comment:
If called without a specific sessionId, signOut needs to signout all sessions (in case of a multi-session app) and then destroy the client.

Do we need to change line 196 as well?